cmd/publicize/server.go

```diff
@@ -102,7 +102,7 @@ func (s *server) handleIssueComment(l *logrus.Entry, ic github.IssueCommentEvent
 		return
 	}
 
-	destOrgRepoLink := fmt.Sprintf("https:///%s/tree/%s", destOrgRepo, baseBranch)
```

Comment:
What about using tests to avoid this kind of issue?

Reply:
I will add more tests.